cassovary-core/src/test/scala/com/twitter/cassovary/graph/MemoryMappedDirectedGraphSpec.scala

```diff
@@ -35,5 +77,50 @@ class MemoryMappedDirectedGraphSpec extends WordSpec with Matchers {
         new MemoryMappedDirectedGraph(new File("nonexistant_file_4398219812437401"))
       }
     }
+
+    " correctly read a list of edges" in {
+      val tempBinaryFile = File.createTempFile("graph1", ".bin")
+      val tempEdgeFile = File.createTempFile("graph1", ".txt")
+      graphToEdgeFormat(testGraph1, tempEdgeFile, false)
+      println(tempEdgeFile.toPath)
+      MemoryMappedDirectedGraph.edgeFileToGraph(tempEdgeFile, tempBinaryFile)
+      val graph1 = new MemoryMappedDirectedGraph(tempBinaryFile)
+      for (testNode <- testGraph1) {
+        val node = graph1.getNodeById(testNode.id).get
+        node.outboundNodes should contain theSameElementsAs (testNode.outboundNodes)
+        node.inboundNodes should contain theSameElementsAs (testNode.inboundNodes)
+      }
+    }
+
+    " correctly read a pair of sorted lists of edges" in {
+      val tempBinaryFile = File.createTempFile("graph1", ".bin")
+      val tempEdgeFile1 = File.createTempFile("graph1_by_id1", ".txt")
+      val tempEdgeFile2 = File.createTempFile("graph1_by_id2", ".txt")
+      graphToEdgeFormat(testGraph1, tempEdgeFile1, false)
```

Comment:
This is somewhat counter-intuitive. Why do we have false and true as opposed to the direction of the edges?